src/data/set6/abilities.ts

```diff
@@ -23,4 +25,25 @@ export default {
 			stunSeconds: champion.getSpellValue('StunDuration'),
 		})
 	},
+	Ezreal: (elapsedMS, spell, champion) => {
+		const doesTargetNearest = true
+		const target = getDistanceUnit(doesTargetNearest, champion)
+		if (!target) {
+			return console.log('No target', champion.name, champion.team)
+		}
+		champion.queueProjectile(elapsedMS, {
+			spell,
+			target,
```

Comment:
We should TODO that Ezreal's ability targets the hex the unit occupied at the time of firing, and does not track, since the engine does not yet support this.